packages/server/src/db/tests/linkController.spec.js

```diff
@@ -10,6 +10,7 @@ describe("test the link controller", () => {
   let table1, table2, appId
 
   beforeEach(async () => {
+    config.modeCloud()
```

Comment:
Can't see any obvious requirement for cloud hosting here, or was the intention something else around multi tenancy?